sdk/messaging/azservicebus/internal/test/test_helpers.go

```diff
@@ -11,7 +11,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/Azure/azure-amqp-common-go/v3/conn"
+	"github.com/Azure/azure-sdk-for-go/sdk/messaging/internal/conn"
 	rm "github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2017-05-10/resources"
```

Comment:
FYI another track 1 dependency (see also next line).

Reply:
Got rid of it now (we're actually not using it!) and that removed a lot of the old stuff.